package.json

```diff
@@ -27,8 +27,12 @@
     "validate-npm-package-name": "^3.0.0"
   },
   "devDependencies": {
+    "@babel/preset-env": "^7.4.4",
```

Comment:
This `package.json` is for `create-nuxt-app` itself instead of the app generated by it.

So I think the dependencies are unnecessary.